troubleshoot/monitoring/node-moves-outages.md

```diff
@@ -9,33 +9,93 @@ products:
+![Move nodes off allocator](images/move_nodes_ech_allocator.jpeg)
 
-**How can I be notified when a node is changed?**
+::::{note}
+You can [configure email notifications](#email) to be alerted when this situation occurs.
+::::
 
-To receive an email when nodes are added or removed from your deployment:
+## Possible causes and impact [ec-node-host-outages]
 
-1. Follow the first five steps in [Getting notified about deployment health issues](../../deploy-manage/monitor/monitoring-data/configure-stack-monitoring-alerts.md).
-2. At Step 6, to choose the alert type for when a node is changed, select **CLUSTER HEALTH** → **Nodes changed** → **Edit alert**.
+Potential causes of system maintenance include, but not limited to, situations like:
 
-::::{note}
-If you have only one master node in your cluster, during the master node vacate no notification will be sent. Kibana needs to communicate with the master node in order to send a notification. One way to avoid this is by shipping your deployment metrics to a dedicated monitoring cluster, which you can configure in Step 2 of [Getting notified about deployment health issues](../../deploy-manage/monitor/monitoring-data/configure-stack-monitoring-alerts.md) when you enable logging and monitoring.
+* A host where the Cloud Service Provider (CSP), like AWS, GCP, or Azure, has notified us in advance about hardware issues.
+* A host that has been abruptly terminated by the Cloud Service Provider (CSP) because of an underlying problem with their infrastructure.
+* Mandatory underlying host OS patch/upgrade activities for security/compliance reasons.
+* Other scheduled maintenance that can be found on the [Elastic status page](https://status.elastic.co/).
+
+Depending on the cause of the node movement, the behavior and expectations differ.
+
+* During planned operations, such as hardware upgrades or host patches, the system attempts to gracefully move the node to another host before shutting down the original one. This process allows shard relocation to complete ahead of time, minimizing any potential disruption.
+
+* In contrast, if a node’s host experiences an unexpected outage, the system automatically vacates the node and displays a related `Don't attempt to gracefully move shards` message on the [activity page](../../deploy-manage/deploy/elastic-cloud/keep-track-of-deployment-activity.md). Since the node is unavailable, the system skips checks that ensure the node’s shards have been moved before shutting down the node.
+
+### Will it cause an outage to my deployment?
+
+Unless overridden or unable, the system will automatically recover the vacated node’s data automatically from replicas or snapshots. If your cluster has [high availability (HA)](https://www.elastic.co/guide/en/cloud/current/ec-planning.html#ec-ha) configured, all search and indexing requests should continue to work within the reduced capacity until the node is replaced.
```

Comment:
```suggestion
Unless overridden or unable, the system will automatically recover the vacated node’s data automatically from replicas or snapshots. If your cluster has [high availability (HA)](https://www.elastic.co/guide/en/cloud/current/ec-planning.html#ec-ha) configured, all search and indexing requests should continue to work within the reduced capacity as the node is replaced.
```

Nit pick, sorry, but technically the search cache has to build up even after the node comes up.